tests/auto/qsparql.pro

```diff
@@ -15,7 +15,7 @@ SUBDIRS = \
     qsparql_virtuoso \
     qsparql_virtuoso_endpoint \
     qsparql_api \
-    qsparql_qmlbindings
+    #qsparql_qmlbindings
```

Comment:
nit: probably does not matter atm, but multiline via escaped eol thing containing "to eol" comment is a bit iffy.

    # SUBDIRS += qsparql_qmlbindings

or stte might be clearer / less accident prone?

Reply:
Sure, can do.